homeassistant/components/binary_sensor/xiaomi_aqara.py

```diff
@@ -203,26 +223,20 @@ def parse_data(self, data, raw_data):
                     self._unsub_set_no_motion()
                 self._unsub_set_no_motion = async_call_later(
                     self._hass,
-                    180,
+                    120,
```

Comment:
120s is the hardware limit / back off period of the motion sensor. This code path applies just to the protocol v2 which doesn't provide a "no_motion" message.